### PR TITLE
avoid pushing empty marker

### PR DESCRIPTION
**Pull request by taikitanaka3**
Remove Route Area Error at containg empty points marker
 
image

### CODE REVIEW

File: map/lanelet2_extension/lib/visualization.cpp [hunk 1]
```diff
@@ -787,9 +787,15 @@ visualization_msgs::msg::MarkerArray visualization::laneletsBoundaryAsMarkerArra
   }
 
   visualization_msgs::msg::MarkerArray marker_array;
-  marker_array.markers.push_back(left_line_strip);
-  marker_array.markers.push_back(right_line_strip);
-  marker_array.markers.push_back(center_line_strip);
+  if (left_line_strip.points.size() > 0) {
```
Comment (kenji-miyake) [inline map/lanelet2_extension/lib/visualization.cpp:790]: `points.size() > 0` -> `!points.empty()`?

Comment (taikitanaka3) [reply map/lanelet2_extension/lib/visualization.cpp]: changed at 594d18b

**Review by kenji-miyake — APPROVED** [review]
I haven't checked the behavior, but the diff looks good to me.

**Comment by taichiH** [comment]
It works fine.
@taikitanaka3 Can you merge?

**Pull request merged**